config/gamemodes.sample.txt

```diff
@@ -21,4 +21,4 @@ PROBABILITY CONSTRUCTION 0
 PROBABILITY CONSPIRACY 0
 PROBABILITY SPY_THEFT 5
 PROBABILITY ARCFIEND 3
-PROBABILITY SALVAGER 0
\ No newline at end of file
+PROBABILITY SALVAGER 2
```

Comment:
This config file is a sample and isn't actually used by the server. Also I broadly agree with Soups that full salvager rounds don't work very well on classic. Maybe with some kind of extra firepower or incentive to go loud early.